### PR TITLE
[sentryexporter] Add exception attributes to extra data

### PR DESCRIPTION
**Pull request by tie**
**Description:**

This PR adds attributes of an exception event to a sentry error event’s extra data map.

**Link to tracking Issue:** n/a

**Testing:** Adjusted test cases for a new function signature.

**Documentation:** n/a

### CODE REVIEW

**Review by codeboten — COMMENTED** [review]
Overall the change looks good to me, will wait for approval from codeowner before approving.

@AbhiPrasad please take a look

File: exporter/sentryexporter/sentry_exporter.go [hunk 1]
```diff
@@ -143,33 +142,34 @@ func convertEventsToSentryExceptions(eventList *[]*sentry.Event, events pdata.Sp
...
 }
 
-// sentryEventFromError creates a sentry event from error event in a span
-func sentryEventFromError(errorMessage, errorType string, span *sentry.Span) (*sentry.Event, error) {
-	if errorMessage == "" && errorType == "" {
-		err := errors.New("error type and error message were both empty")
-		return nil, err
+// sentryEventFromError converts exception attributes of an error event in
+// a given span to a sentry exception event.
+func sentryEventFromError(attrs pdata.AttributeMap, span *sentry.Span) *sentry.Event {
```
[the remark this answers is not in the document]
Comment (AbhiPrasad) [reply exporter/sentryexporter/sentry_exporter.go:155]: Yes I agree it's what we should use.

**Review by AbhiPrasad — APPROVED** [review]
Looks good to me, thank you for your contribution!

File: exporter/sentryexporter/sentry_exporter.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (tie) [reply exporter/sentryexporter/sentry_exporter.go]: Sure, I did a `:GoRename` on the declaration.

**Comment by tie** [comment]
@AbhiPrasad hi! The bot closed the PR for some reason. Is there anything else I should change before we can merge this?

**Comment by AbhiPrasad** [comment]
Think it looks good, needs approval from the collector maintainers now. @owais?

**Pull request closed without merging**